security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,6 +12,57 @@ These release notes track the development of {cert-manager-operator}.
 
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
+[id="cert-manager-operator-release-notes-1-15-0"]
+== {cert-manager-operator} 1.15.0
+
+Issued: 2025-01-22
+
+The following advisory are available for the {cert-manager-operator} 1.15.0:
+
+* link:https://access.redhat.com/errata/RHEA-2025:0487[RHEA-2025:0487]
+* link:https://access.redhat.com/errata/RHSA-2025:0535[RHSA-2025:0535]
+* link:https://access.redhat.com/errata/RHSA-2025:0536[RHSA-2025:0536]
+
+Version `1.15.0` of the {cert-manager-operator} is based on the upstream cert-manager version `v1.15.4`. For more information, see the link:https://cert-manager.io/docs/releases/release-notes/release-notes-1.15/#v1154[cert-manager project release notes for v1.15.4].
+
+=== New features and enhancements
+
+*Scheduling overrides for {cert-manager-operator}*
+
+With this release, you can configure scheduling overrides for {cert-manager-operator}, including the cert-manager controller, webhook, and CA injector.
+
+For more information, see xref:../../security/cert_manager_operator/cert-manager-customizing-api-fields.adoc#cert-manager-scheduling-overrides[Configuring scheduling overrides for {cert-manager-operator}].
+
+*Google CAS issuer*
+
+The {cert-manager-operator} now supports the Google Certificate Authority Service (CAS) issuer. The `google-cas-issuer` is an external issuer for cert-manager that automates certificate lifecycle management, including issuance and renewal, with CAS-managed private certificate authorities.
+
+[NOTE]
+====
+The Google CAS issuer is validated only with version 0.9.0 and {cert-manager-operator} version 1.15.0. These versions support tasks such as issuing, renewing, and managing certificates for the API server and ingress controller in {product-title} clusters.
+====
+
+*Default installMode updated to `AllNamespaces`*
+
+Starting from version 1.15.0, the default and recommended Operator Lifecycle Manager (OLM) installMode is AllNamespaces. Previously, the default was SingleNamespace. This change aligns with best practices for multi-namespace operator management.
```

Comment:
Should these instances of installMode be in backticks? I feel like they should be if they were styled that way in your other PR:
```suggestion
*Default `installMode` updated to `AllNamespaces`*

Starting from version 1.15.0, the default and recommended Operator Lifecycle Manager (OLM) `installMode` is AllNamespaces. Previously, the default was SingleNamespace. This change aligns with best practices for multi-namespace operator management.
```